### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by mihuan-0**
Add Chinese translation (Simplified and Traditional Chinese) for all stringtable.xml.

### CODE REVIEW

**Comment by neilzar** [comment]
Why did you close the previous one? This one seems to be identical apart from a bunch of indentation errors that were introduced.

**Comment by mihuan-0** [comment]
At first, I downloaded the file and change on my computer three days ago. Today I downloaded the latest file again and compared the files via Beyond Compare. It is possible that some indentation errors occurred in the meantime.

File: addons/context_actions/stringtable.xml [hunk 1]
```diff
@@ -38,100 +44,130 @@
...
+            <Chinesesimp>随意开火</Chinesesimp>
+            <Chinese>隨意開火</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_FireAtWillEngage">
             <English>Fire &amp; Engage At Will</English>
             <German>Feuerfreigabe und selbstständiger Angriff</German>
             <Korean>사격 &amp; 자유 사격 개시</Korean>
             <Polish>Ogien i samowolona inicjacja</Polish>
             <Japanese>随意射、命令時射撃</Japanese>
+            <Chinesesimp>随意开火 &amp; 随意反击</Chinesesimp>
```
Comment (Kexanone) [inline addons/context_actions/stringtable.xml:142]: `&amp;` is just the HTML code for an ampersand.

Comment (mihuan-0) [reply addons/context_actions/stringtable.xml]: Thanks for the reminder, but the question at hand is very confusing to me. It suggests that there are three files with space indentation errors. But everything is correct when I look at the files both in Github and VSCode.

Comment (Kexanone) [reply addons/context_actions/stringtable.xml]: If you [enable commits from maintainers](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork), we could just quickly run a script over it to fix it 😉

Comment (mharis001) [reply addons/context_actions/stringtable.xml]: You have tabs instead of spaces in those files. We use 4 spaces per indentation level.

Comment (mihuan-0) [reply addons/context_actions/stringtable.xml]: I enable allow edits by maintainers. Not sure if it's the same function。
I have once again rewritten the wrong line and the problem should now be solved, hopefully！

File: addons/editor/stringtable.xml [hunk 2]
```diff
@@ -162,96 +200,127 @@
...
             <English>Sets focus to the search bar.</English>
             <German>Setzt den Fokus auf das Suchfeld.</German>
             <Polish>Kieruje na listwe wyszukiwarki</Polish>
             <Japanese>検索バーにフォーカスします。</Japanese>
+            <Chinesesimp>指向搜索栏</Chinesesimp>
+            <Chinese>指向搜索欄</Chinese>
         </Key>
         <Key ID="STR_ZEN_Editor_OrientTerrainNormal_Description">
             <English>Orients all selected objects to match the terrain slope at their position.</English>
-            <German>Orientiert alle ausgewählten Objekte der Steigung an ihrer Position.</German>
```
Comment (mharis001) [inline addons/editor/stringtable.xml:230]: Accidently removed?

File: addons/faction_filter/stringtable.xml [hunk 3]
```diff
@@ -6,6 +6,8 @@
             <French>Zeus Enhanced - Filtre des Fractions</French>
             <German>Zeus Enhanced - Fraktionsfilter</German>
             <Japanese>Zeus Enhanced - 陣営フィルタ</Japanese>
+            <Chinesesimp>Zeus增强--阵营筛选</Chinesesimp>
```
Comment (mharis001) [inline addons/faction_filter/stringtable.xml:9]: I think keep "Zeus Enhanced" untranslated.

File: addons/editor/stringtable.xml (hunk 2 above)
Comment (mihuan-0) [reply addons/editor/stringtable.xml:230]: Sorry, maybe. Fixed.

File: addons/faction_filter/stringtable.xml (hunk 3 above)
Comment (mihuan-0) [reply addons/faction_filter/stringtable.xml]: ok

**Review by mharis001 — APPROVED** [review]

**Comment by mharis001** [comment]
Thank you for translating.

File: addons/tasks/stringtable.xml [hunk 4]
```diff
@@ -5,55 +5,75 @@
...
             <Japanese>タスクアイコンを設定します。</Japanese>
+            <Chinesesimp>设置任务图标</Chinesesimp>
+            <Chinese>設置任務圖標</Chinese>
         </Key>
         <Key ID="STR_ZEN_Tasks_TitleDescription">
             <English>Title &amp; Description</English>
             <German>Titel &amp; Beschreibung</German>
             <Japanese>タイトル &amp; 概要</Japanese>
+            <Chinesesimp>标题&amp;描述</Chinesesimp>
+            <Chinese>標題&amp;描述</Chinese>
```
Comment (Kexanone) [inline addons/tasks/stringtable.xml:37]: Another `&amp;`

**Review by Kexanone — CHANGES_REQUESTED** [review]

File: addons/context_actions/stringtable.xml (hunk 1 above)
Comment (Kexanone) [reply addons/context_actions/stringtable.xml]: Yes, it's the correct one, but I see you solved it on your own in the meantime 🥳

File: addons/tasks/stringtable.xml (hunk 4 above)
Comment (mihuan-0) [reply addons/tasks/stringtable.xml]: ok

**Review by Kexanone — APPROVED** [review]

**Pull request merged**